TestMappings.json

```diff
@@ -161,6 +161,9 @@
   "src/ResourceManager/Scheduler/": [
     ".\\src\\ResourceManager\\Scheduler\\Commands.Scheduler.Test\\bin\\Debug\\Microsoft.Azure.Commands.Scheduler.Test.dll"
   ],
+  "src/ResourceManager/Search/": [
```

Comment:
This should be:
".\\src\\ResourceManager\\Search\\Commands.Management.Search.Test\\bin\\Debug\\Microsoft.Azure.Commands.Management.Search.Test.dll"

Or you could remove the "management" from the folder name.

Reply:
Yeah...thanks for catching this!